tests/integration/ha/conftest.py

```diff
@@ -14,6 +14,14 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture
```

Comment:
This does not need to be a fixture, it will limit us in the usage when we support dual base

Reply:
So, I added as a fixture exactly thinking of dual base: a single function that will decide prior to any `build_and_deploy...`  calls, which charm will we build.